conda_build/render.py

```diff
@@ -934,7 +934,7 @@ def open_recipe(recipe: str | os.PathLike | Path) -> Iterator[Path]:
     elif recipe.suffixes in [[".tar"], [".tar", ".gz"], [".tgz"], [".tar", ".bz2"]]:
         # extract the recipe to a temporary directory
         with TemporaryDirectory() as tmp, tarfile.open(recipe, "r:*") as tar:
-            tar.extractall(path=tmp)
+            tar.extractall(path=tmp, filter="data")
```

Comment:
This happens when we render from an existing tarball package. The only file we care about in this scenario is the recipe itself so AFAIK removing group and other's write and execute bits is safe to do.

Reply:
this thread is moot since we delegate to CPH/CPS